contracts/FFPaymentSplit.sol

```diff
@@ -23,6 +29,8 @@ contract FFPaymentSplit is PaymentSplitter {
 
     // Mapping between addresses and how much money they have withdrawn.
     mapping(address => uint) public amountsWithdrew;
```

Comment:
I haven't cleaned up these variables yet, since I'm not sure if you think what I have in this PR is a better approach, or that you'd still like to keep these around regardless.